src/ui/settings/ViewNetworkSettings.qml

```diff
@@ -69,7 +69,7 @@ VPNFlickable {
         anchors.top: localNetwork.visible ? localNetwork.bottom : ipv6.bottom
         visible: !vpnFlickable.vpnIsOff
 
-        //% "VPN must be off to edit network settings"
+        //% "VPN must be off to edit settings"
```

Comment:
For the future, please use new IDs when making these changes. If you don't, you'll have strings in localized files that don't match the en-US source ([more info on the general process here](https://mozilla-l10n.github.io/documentation/localization/making_string_changes.html)).

The infra in the l10n repository will invalidate these translations on the next merge, but in the meantime you're in a limbo.